go/vt/vttls/crl.go

```diff
@@ -83,7 +82,7 @@ func loadCRLSet(crl string) ([]*pkix.CertificateList, error) {
 			continue
 		}
 
-		parsedCRL, err := x509.ParseCRL(block.Bytes)
+		parsedCRL, err := x509.ParseRevocationList(block.Bytes)
```

Comment:
`PEM` decoding here is already done explicitly so there's no user facing changes in the usage (still only `PEM` formatted CRLs are supported).